packages/codemirror-extension/schema/commands.json

```diff
@@ -45,6 +45,24 @@
       "title": "Ctrl-C",
       "description": "When enabled, which is the default, doing copy or cut when there is no selection will copy or cut the whole lines that have cursors on them.",
       "default": true
+    },
+    "toggleComment": {
+      "type": "string",
+      "title": "Block Comment",
+      "description": "Key command for block comment",
+      "default": "Ctrl-/"
+    },
+    "toggleCommentMac": {
+      "type": "string",
+      "title": "Block Comment on MacOS keyboard",
+      "description": "Key command for block comment with MacOS",
+      "default": "Cmd-/"
```

Comment:
Does JupyterLab support using `Accel` in this file to avoid having two different shortcuts (if that is a benefit)?

Reply:
I think so, but I just re-interpreted what jlab was already doing into being settings:
https://github.com/jupyterlab/jupyterlab/blob/14776bb75832c0a474933dc46aba67cda16ced9e/packages/codemirror/src/factory.ts#L40-L41

Is accel = `ctrl` or `alt`?

Reply:
Ok if this is what JupyterLab has elsewhere then just keep it as is I guess unless someone else with more knowledge chimes in. I believe `Accel` is set to `Ctrl` on Win/Linux and `Cmd` on macOS.